python/dlisio/plumbing/channel.py

```diff
@@ -101,6 +101,25 @@ def __init__(self, obj = None, name = None):
         self._fmtstr       = None
         self.frame        = None
 
+    @property
+    def index(self):
+        """The Channel this Channel is indexed against, if any. See
+        :attr:`Frame.index_type` for definition of existing index channel.
+
+        Returns
+        -------
+
+        channel : Channel or None
+        """
+        msg = 'This channel is an index channel'
+        index = self.frame.index
+
+        if index == self:
```

Comment:
Why would you do that, it is a clear violation of the standard!? Also, creating a new channel object with the same name, origin, copy is _not_ enough to parse the curve, so it would not be a valid index channel. So no, I do not think we should redefined object comparison